_inc/client/plans/plan-body.jsx

```diff
@@ -64,43 +68,131 @@ const PlanBody = React.createClass( {
+									this.props.isPluginInstalled( 'vaultpress/vaultpress.php' )
+									&& this.props.isPluginActive( 'vaultpress/vaultpress.php' ) ? (
+										<Button href="https://vaultpress.com/" className="is-primary">
+											{ __( 'TO DO: Text and link if unconfigured' ) }
+										</Button>
+									)
+									: (
+										<Button href={ 'https://videopress.com' + this.props.siteRawUrl + '?only=vaultpress' } className="is-primary">
+											{ __( 'TO DO: Text and link if unconfigured' ) }
+										</Button>
+									)
+								}
+							</div>
+						: ''
+					}
+
+					{
+						includes( [ 'jetpack_business', 'jetpack_business_monthly' ], this.props.plan ) ?
+							<div className="jp-landing__plan-features-card">
+								<h3 className="jp-landing__plan-features-title">{ __( 'Surveys & Polls' ) }</h3>
+								<p>{ __( 'Unlimited surveys, unlimited responses. Use the survey editor to create surveys quickly and easily. Collect responses via your website, e-mail or on your iPad or iPhone.' ) }</p>
+								<Button href="https://polldaddy.com/dashboard/" className="is-primary">
+									{ __( 'Create a new poll' ) }
+								</Button>
+							</div>
+							: ''
+					}
+
+					{
+						includes( [ 'jetpack_personal', 'jetpack_personal_monthly' ], this.props.plan ) ?
+							<div className="jp-landing__plan-features-card">
+								<h3 className="jp-landing__plan-features-title">{ __( 'Need more? Running a business site?' ) }</h3>
+								<p>{ __( 'If your site is important to you, consider protecting and improving it with some of our advanced features: ' ) }</p>
+								<p> &mdash; { __( 'Daily and on-demand security scanning' ) }</p>
+								<p> &mdash; { __( 'Real-time backups and one-click threat resolution' ) }</p>
+								<p> &mdash; { __( 'Unlimited and ad-free video hosting' ) }</p>
+								<p> &mdash; { __( 'Advertising revenue with WordAds' ) }</p>
+								<p> &mdash; { __( 'Advanced polls and ratings' ) }</p>
+								<p>
+									<Button href={ 'https://wordpress.com/plans/' + this.props.siteRawUrl } className="is-primary">
+										{ __( 'Compare Plans' ) }
+									</Button>
+								</p>
+							</div>
+						: ''
+					}
+
+					{
+						includes( [ 'jetpack_premium', 'jetpack_premium_monthly' ], this.props.plan ) ?
+							<div className="jp-landing__plan-features-card">
+								<h3 className="jp-landing__plan-features-title">{ __( 'Need more? Running a business site?' ) }</h3>
+								<p>{ __( 'If your site is important to you, consider protecting and improving it with some of our advanced features: ' ) }</p>
+								<p> &mdash; { __( 'On-demand security scanning' ) }</p>
+								<p> &mdash; { __( 'Real-time backups' ) }</p>
+								<p> &mdash; { __( 'One-click threat resolution' ) }</p>
+								<p> &mdash; { __( 'Advanced polls and ratings' ) }</p>
+								<p>
+									<Button href={ 'https://wordpress.com/plans/' + this.props.siteRawUrl } className="is-primary">
```

Comment:
@richardmuscat What's the reason for using paragraph tags and em dashes instead of just implementing an UL?